test/unit-test/README.md

```diff
@@ -33,7 +33,7 @@ Go to the root directory of the FreeRTOS+TCP repo and run the following script:
 #!/bin/bash
 # This script should be run from the root directory of the FreeRTOS+TCP repo.
 
-if [[ ! -f FreeRTOS_IP.c ]]; then
+if [[ ! -d source ]]; then
     echo "Please run this script from the root directory of the FreeRTOS+TCP repo."
     exit 1
 fi
```

Comment:
Non-Blocking: Why not just make this as an actual shell script in the root directory?

Reply:
@kstribrnAmzn  Not sure if this is something that frequently used/required by the normal users.

Reply:
I personally agree with the idea of making these actual scripts that live in the repo instead of something that needs to be copy/pasted. But I think doing this is a standard across a few of our repos. Additionally, I'm not sure if we want to make a decision to add it as a file as part of this PR.